bazel/repository_locations.bzl

```diff
@@ -8,9 +8,9 @@ REPOSITORY_LOCATIONS = dict(
         urls = ["https://mirror.bazel.build/github.com/bazelbuild/bazel-skylib/releases/download/1.0.3/bazel-skylib-1.0.3.tar.gz"],
     ),
     com_envoyproxy_protoc_gen_validate = dict(
-        sha256 = "c695fc5a2e5a1b52904cd8a58ce7a1c3a80f7f50719496fd606e551685c01101",
-        strip_prefix = "protoc-gen-validate-0.6.1",
-        urls = ["https://github.com/envoyproxy/protoc-gen-validate/archive/refs/tags/v0.6.1.tar.gz"],
+        sha256 = "27b3202ba6e9eb37dca0f902c8f96b6330670b33d8669ad739f60ab61bdaf296",
+        strip_prefix = "protoc-gen-validate-1.0.2",
+        urls = ["https://github.com/envoyproxy/protoc-gen-validate/archive/refs/tags/v1.0.2.tar.gz"],
```

Comment:
I aligned protoc-gen-validate with it's go.md version, see : 
https://github.com/cncf/xds/pull/73/files#diff-340a0a599142950de3cef119fa68e93ef2350dbab23edaa3c7e561899e9ea399R6